UI.PageModels/UI.PageModels.csproj

```diff
@@ -6,9 +6,13 @@
         <Nullable>enable</Nullable>
 
         <NoWarn>csharpsquid:S1699</NoWarn>
+
+        <UserSecretsId>9efcb29f-fb51-4a48-b487-572ada7d6435</UserSecretsId>
```

Comment:
Make sure this is the same secrets id as the other projects else we need to manage multiple secret files for one repo